vendor/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -168,7 +168,7 @@ type MemoryStats struct {
 	// dirty memory, and kernel memory. WorkingSetBytes is <= UsageBytes
 	// +optional
 	WorkingSetBytes *uint64 `json:"workingSetBytes,omitempty"`
-	// The amount of anonymous and swap cache memory (includes transparent
+	// The amount of anonymous memory (includes transparent
```

Comment:
> contains memory excluding cached 

So we need to fix this vendor file, right ?

> Why would you change this vendor file without updating godeps.

How / Where do I update the godeps ? sorry I'm new to this.